oauth2/authorize.php

```diff
@@ -56,6 +59,13 @@
 }
 
 $end_point = $gbl::getRequestEndPoint();
+$logger->debug("oauth2 endpoint is " . $end_point);
```

Comment:
if this debug is gonna get to the php error log (I am guessing, then should escape the variable:
```
$logger->debug("oauth2 endpoint is " . errorLogEscape($end_point));
```

Reply:
I modified the logger internals so that anything passed as a context variable is auto escaped.